compiler/compiler.go

```diff
@@ -565,10 +565,10 @@ func (c *CompilationCache) Compile(in semantic.MonoType) (Func, error) {
 	panic("unimpleneted")
 }
 
-type funcErr struct {
-	F   Func
-	Err error
-}
+//type funcErr struct {
```

Comment:
A lot of the compilation cache code around this is commented out, so I just followed suit here.

Reply:
I recommend we just delete the code instead of commenting it out.